addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -227,6 +236,63 @@ def _rtc_join_call(self, check_rtc_session_ids=None):
                 res['invitedMembers'] = [('ADD', list(invited_members._discuss_channel_member_format(fields={'id': True, 'channel': {}, 'persona': {'partner': {'id', 'name', 'im_status'}, 'guest': {'id', 'name', 'im_status'}}}).values()))]
         return res
 
+    def _join_sfu(self):
+        if len(self.channel_id.rtc_session_ids) < SFU_MODE_THRESHOLD:
+            if self.channel_id.sfu_channel_uuid:
+                self.channel_id.sfu_channel_uuid = None
+                self.channel_id.sfu_server_url = None
+            return
+        elif self.channel_id.sfu_channel_uuid:
```

Comment:
What if there is a `sfu_channel_uuid` but no `sfu_server_url` on the channel?
Does it use the one from the config, or it will fail?

Reply:
without a `sfu_server_url`, clients will not get the rtc server info* and will stay/use the peer to peer system.

\* https://github.com/odoo/odoo/pull/132153/files/f55716f8403f7eb68d3d5ca105060c4c4e7a866d#diff-cd763b90c80860ef917a5ec3a7ad2f2ec9dfd4a8067b5514d1d16b64c36d1696R284-R285

Reply:
Yes it assumes that both are set and cleared together, but I will check them both here to be sure